smalltalksrc/Slang/TMethod.class.st

```diff
@@ -916,6 +925,15 @@ TMethod >> ensureConditionalAssignmentsAreTransformedIn: aCodeGen [
 	 and: [self replaceNodesIn: transformedAssignments.
 		  true]] whileTrue
 		
+]
+
+{ #category : 'inlining' }
+TMethod >> ensureIsValidMethodAfterInlining: aNode in: aCodeGen [
+	"inlining can cause some issues in the AST that will produce incorrect code, modify the AST if needed to produce a cleaner one. Return true if anything was done "
```

Comment:
This comment can be more precise too. Otherwise we don't know what really is happenning. If the comment is not precise, then we can just remove it.

Another thing here, this method is called `ensureIsValidMethodAfterInlining:in:` and just calls this `iterateThrough`. Do we really need both?